test/e2e/explainer/test_aix_explainer.py

```diff
@@ -74,7 +74,8 @@ def test_tabular_explainer():
         logging.info(KFServing.api_instance.get_namespaced_custom_object("serving.knative.dev", "v1",
                      KFSERVING_TEST_NAMESPACE, "services", service_name + "-predictor-default"))
         pods = KFServing.core_api.list_namespaced_pod(KFSERVING_TEST_NAMESPACE,
-                                                      label_selector='serving.kubeflow.org/inferenceservice={}'.format(service_name))
+                                                      label_selector='serving.kubeflow.org/inferenceservice={}'
+                                                      .format(service_name))
```

Comment:
no new line here as it breaks the e2e test
```suggestion
                                                      label_selector='serving.kubeflow.org/inferenceservice={}'.format(service_name))
```